src/api/app/helpers/webui/user_helper.rb

```diff
@@ -1,10 +1,11 @@
 module Webui::UserHelper
+  # This method is migrated to Webui2 (and refactored) with the name: image_tag_for
   # @param [User] user object
   def user_image_tag(user, opt = {})
```

Comment:
If we start using this for groups too, we should use `user_or_group` instead of `user`.

Reply:
this is the old UI, we are going to drop it. We just added here so that the staging workflow is consistent and that we don't forget when migrating (the comment is there to remember you to use the correct method when migrating)

Reply:
Oh alright.